### ISSUE TITLE
More videos than the json file presents

### ISSUE THREAD

**Issue by rightly0716** [hangzhaomit/HACS-dataset#13]
I would like to thank the authors for the great contribution of annotated videos/clips!

I see something weird, and not sure whether I understand correctly. I downloaded the videos and obtained the missing ones following instruction on the README.txt. Also I downloaded the json file which stores all the timestamp information of each video. 

Somehow, I find the number of videos is much larger than what is annotated in json. For example, the folder "Drinking_beer" contains 1485 videos (mp4) in my download file, but the json file only has 202 titles that contain "Drinking_beer" in 'label'. This is for training set. 

Do I understand the logic correctly? Or only a proportion of training videos (i.e., 202/1485) titles are annotated? 

I noticed that the json file only contains 49581 rows. I guess maybe the video folder contains both video and clip, with the same folder name (for example "Drinking_beer") but different video_id names? 

====Parsing clips====
[training set]: 492748 videos, 1509478 clips
[validation set]: 5981 videos, 20245 clips
[testing set]: 5987 videos, 20293 clips
====Parsing segments====
[training set]: 37613 videos
[validation set]: 5981 videos
[testing set]: 5987 videos
Done.

**Comment by hangzhaomit** [comment]
Note that the .csv file is annotation of clips, and .json file is annotation of segments. We have much less segment annotations than clip annotations.

**Comment by rightly0716** [comment]
Got it! And I guess the clips are cut from the segments right? Thanks a lot for your timely reply! 

I think when I ran the script to download the videos/clips, they were placed under the same directory. Is there a quick way to tell which ones are segments and which ones are clips, from their names?

**Comment by hangzhaomit** [comment]
Yes they are placed in the same directory. Segment videos is a subset of Clip videos.  Maybe you can choose to download segments only.

**Comment by rightly0716** [comment]
Got it. Thanks a lot!
